ui.R

```diff
@@ -5,101 +5,115 @@ airports <- airportr::airports %>%
+        )
+      )
+    )),
     fluidRow(column(
       6, actionButton(inputId = "go", label = "Go!"),
     )),
+    # float emissions over map ----
     fluidRow(
-      shiny::HTML("<div align='center'>"),
-      htmlOutput("emissions"),
-      shiny::HTML("</div>")
+      column(6,
+      shiny::HTML("<div class='floatcontainer'>"),
+          shiny::HTML("<div class='box' style='background: black;'>"),
+            leaflet::leafletOutput("map"),
+          shiny::HTML("</div>"), #box div
+          shiny::HTML("<div class='box stack-top' style='background: white;'>"),
+            htmlOutput("emissions"),
+          shiny::HTML("</div>"), #stack-top div
+      shiny::HTML("</div>")), #container div
+      column(6,
+             p(htmlOutput("carbon_budget")),
+             p(htmlOutput("ice_melt")),
+             p(htmlOutput("trees")),
+             )
     ),
-    fluidRow(leaflet::leafletOutput("map")),
-    fluidRow(column(
-      6,
+    fluidRow(
       shiny::p("* All estimates include radiative forcing")
-    ))
+    ),
+    box(
+      title = "Methodology", solidHeader = TRUE,
+      collapsible = TRUE,
+      collapsed = TRUE,
+      width = 10,
+      h3("Emissions"),
+      HTML(
+        paste(
+          "Emissions are calculated with the <a href='https://github.com/acircleda/footprint' target='_blank'>footprint R package</a>.")),
+      h3("Personal Carbon Footprint"),
+      HTML("The personal carbon footprint for 2020-2030 ranges from 3.7 to 4.8 metric tons of CO2e per person. This range is dervived from a number of scenarios that limit global warming to 1.5 or 2 degrees Celsius above pre-industrial temperatures. For our calculator's estimate, we divide the metric tons of CO2e from the inputted trip by the upper range of the carbon budget, 4.8.",
+           "<br><br>Source: <a href='https://www.iges.or.jp/en/pub/15-degrees-lifestyles-2019/en' target='_blank'>1.5-Degree Lifestyles: Targets and options for reducing lifestyle carbon footprints</a>"),
+      h3("Arctic Ice Melt"),
+      HTML("This estimate is based on a linear relationship between CO2e and arctice ice melt at a rate of approximately 3 square meters per metric ton of CO2",
+           "<br><br>Source: <a href='https://doi.org/10.1126/science.aag2345' target='_blank'>Observed Arctic sea-ice loss directly follows anthropogenic CO2 emission</a>"),
+      h3("Tree Planting"),
+      HTML("The number of newly-planted trees required to offset a trip specifically refers to the number of urban medium growth coniferous or deciduous tree seedlings grown for 10 years in the United States. It is estimated that one tree in this scenario can sequester .060 metric tons of CO2.",
+           "<br><br>Source: <a href='file:///C:/Users/aschmidt/Zotero/storage/2YEFXCII/greenhouse-gases-equivalencies-calculator-calculations-and-references.html' target='_blank'>Greenhouse Gases Equivalencies Calculator - Calculations and References</a>"),
+    )
```

Comment:
I'm not sure if this part displays correctly on my screen, but this is how it appears. Is that expected?
![Screenshot 2021-07-21 at 16 10 49](https://user-images.githubusercontent.com/7890048/126514959-52a9e8da-31e5-41d2-b40f-6bf309bf87e1.png)

Reply:
This is how it appears for me, too. I haven't worked on a fix yet.